### ISSUE TITLE
Alchemist allowing for book upgrade using lvl 1 books.

### ISSUE THREAD

**Issue by markusew** [GC-spigot/AdvancedEnchantments#533]
Hey, there's a bug making possible to upgrade an enchant book using a lvl 1 book togheter with any other lvl book. For example you can upgrade a book to lvl 3 by using a lvl 1 + lvl 2 book. 

This is done by putting the lvl 1 book in the first slot in the alchemist and then the higher lvl book in the second slot. This will give you a upgraded book.

**Comment by GC-spigot** [comment]
@markusew what plugin version are you using?

**Comment by markusew** [comment]
> @markusew what plugin version are you using?

5.7.4 on 1.13.2

**Comment by GC-spigot** [comment]
Alright I will try on 1.13.2

**Comment by markusew** [comment]
If it doesn't work I'll provide more information

**Comment by GC-spigot** [comment]
http://n3.fyi/cant.gif

**Comment by markusew** [comment]
https://streamable.com/0seve

```alchemist:
  settings:
    combine-books: true
    combine-dust: true
  books:
    add-up-chances: true
    pre-set-chances:
      enabled: false
      success: 50
      destroy: 50
  dusts:
    add-up-and-split: true
    add-up: false
  price:
    exp: false
    let-plugin-count: true
    pre-set-pricing:
      enabled: true
      price: 1000
    let-plugin-decide: false
  messages:
    inv-name: Alchemist
    item-preview:
      name: '&e&lItem Preview'
      lore:
      - '&7A preview of item you will'
      - '&7receive from the Alchemist'
      - '&7will be displayed here.'
      item:
        type: STAINED_GLASS_PANE
        damage: 0
    item-explanation:
      name: '&e&lAlchemist'
      lore:
      - '&7The Alchemist will Exchange...'
      - ''
      - '&b&l* &f2X Enchantment Books'
      - '&7(of the same type and level)'
      - '&d= &f1X Enchantment Book'
      - '&7(of higher level)'
      item:
        type: STAINED_GLASS_PANE
        damage: 14
    item-combine:
      name: '&b&lCLICK TO EXCHANGE'
      lore:
      - '&c&lCOSTS &a&l$%price%'
      - ''
      - '&7Click here to confirm this'
      - '&7item exchange and recieve your'
      - '&7epic new item shown above!'
      item:
        type: STAINED_GLASS_PANE
        damage: 5
    item-placeholder:
      name: ' '
      lore: []
      item:
        type: STAINED_GLASS_PANE
        damage: 15
    doesnt-accept: '&cAlchemist doesn''t accept this item.'```

**Comment by GC-spigot** [comment]
Fixed

**Comment by markusew** [comment]
I am not sure what you changed but I am now unable to set a custom price for alchemist without having this bug.

Perhaps I am configuring it wrong but I feel like I've tested all possibilities. I want to be able to set the cost for any book combining to $1000. Right now it's $2000 x lvl of books. 

If I change the setting: "let-plugin-count:" to false I am able to change the price but then the bug I reported still exists.

Any help with this issue would be greatly appreciated. Thanks in advance.
